source/databricks/calculation-engine/tests/calculation_input/test_charges_reader.py

```diff
@@ -54,7 +59,7 @@ def _create_charge_master_data_row(
         Colname.from_date: from_date,
         Colname.to_date: to_date,
     }
-    return row
+    return Row(**row)
```

Comment:
In conjuction with previous suggestion
```suggestion
    return row
```

Reply:
Applies to other occurrences too.